src/components/externalLinks/externalLinks.component.scss

```diff
@@ -29,7 +29,7 @@
         img {
             width: 16px;
             height: 16px;
-            margin-right: 12px;
+            margin-right: 8px;
```

Comment:
Why using this instead can use inline classname as `icon-dim-16 mr-8`

Reply:
Done